src/request.rs

```diff
@@ -132,6 +132,12 @@ macro_rules! lsp_request {
     ("callHierarchy/outgoingCalls") => {
         $crate::request::CallHierarchyOutgoingCalls
     };
+    ("typeHierarchy/supertypes") => {
+        $crate::request::TypeHierarchySupertypes
+    };
+    ("typeHierarchy/subtypes") => {
```

Comment:
nit: these should be sorted after the `textDocument/*` items

Reply:
Thanks so much for the approval! I'll fix that right away in another commit.

Reply:
Done in d602cd1!

Reply:
no big deal but IMO it's better to squash this into the original commit, eradicating the mistake for the benefit of future readers

Reply:
No worries! I can squash this commit into 37ec22c and force-push.

Reply:
Done!